templates/agent/puppet.conf.erb

```diff
@@ -33,3 +33,9 @@
     noop              = <%= scope.lookupvar('::puppet::agent_noop') %>
     configtimeout     = <%= scope.lookupvar('::puppet::configtimeout') %>
     usecacheonfailure = <%= scope.lookupvar('::puppet::usecacheonfailure') %>
+<% if scope.lookupvar('::puppet::prerun_command') and !scope.lookupvar('::puppet::prerun_command').to_s.empty? -%>
+    prerun_command    = <%= scope.lookupvar('::puppet::prerun_command') %>
```

Comment:
not really, but if the first check is false, the second doesn't get evaluated (at least I'm not seeing any errors)